cli/flox-rust-sdk/src/models/environment/generations.rs

```diff
@@ -839,7 +839,7 @@ pub struct SingleGenerationMetadata {
     pub created: DateTime<Utc>,
 
     /// unix timestamp of the time when this generation was last set as live
-    /// `None` if this generation has never been set as live
+    /// `None` if this generation is currently live
```

Comment:
@ysndr is it possible for a generation to never be live? In which case it would be None if either the generation has never been live or is currently live?